### DISCUSSION TITLE
5HQxJVRi8Q4cjGgaqn2SBEnDfcSr3Vt33fNhDUkSwZwbVRCf

### DISCUSSION THREAD

**Discussion by nonboring** [entropyxyz/community#22] [Get Test Funds] [answered]
### Address

5HQxJVRi8Q4cjGgaqn2SBEnDfcSr3Vt33fNhDUkSwZwbVRCf

**Comment by johnnymatthews** [comment, marked as answer]
Sent!

**Reply by nonboring** [reply]
Hi! I still have not received any tokens. Shall I do anything else to get something on my balance?
<img width="626" alt="image" src="https://github.com/entropyxyz/community/assets/114029093/5302e04f-40f8-4430-962a-64477ce3a6af">

**Reply by johnnymatthews** [reply]
It looks like the transfer function within the CLI isn't working as expected. We're looking into this now.
